domains/oliik2013.thedev.me.json

```diff
@@ -6,7 +6,7 @@
   "description": "domain for me yay for now for website",
 
   "records": {
-    "CNAME": "oliik2013.github.io"
+    "CNAME": [oliik2013.github.io]
```

Comment:
Add it as a string i.e. ["oliik2013.github.io"]

Reply:
i changed it